src/clash.rs

```diff
@@ -85,6 +85,10 @@ impl Clash {
         self.last_version.data.constraints.as_deref()
     }
 
+    pub fn stub_generator(&self) -> Option<&str> {
```

Comment:
This is confusing, the clash doesn't have a stub_generator, it has a stub that then goes through the cg stub_generator to produce the default code. 

From https://www.codingame.com/playgrounds/40701/help-center/stub-generator:

> The stub generator is the program which will write in every available language the default code you get when you open the IDE. To do this, the generator needs input written using the syntax below.

So I think here it should really just be `data.stub.as_deref()` etc.